### DISCUSSION TITLE
[v3] js error wire:model directly to model attribute

### DISCUSSION THREAD

**Discussion by jesperbjoernandersen** [livewire/livewire#6336] [7. Bugs (archived)]
### Livewire version

v3

### Which PHP version are you using?

PHP 8.2

### Steps To Reproduce

1. Have a component with a $user property that gets a user model assigned to in in the mount() method
2. Make an input field in the view that does wire:model="user.name"
3. Input correctly shows the users name, but a console error is thrown, saying cannot read properties of null (reading "name")

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by devront-it** [comment]
Solved it by wrapping the input into @if($user) ...your input here @endif
But I also think this is not expected behaviour.

**Comment by JuanDeLeon** [comment]
V3 does not support model-property binding by default, you need to set 'legacy_model_binding' => true, in Livewire config file.
https://livewire.laravel.com/docs/upgrading#eloquent-model-binding
